test/performance/commCompare/gbt/many-to-one.compopts

```diff
@@ -0,0 +1,4 @@
+-sop=opGet -snumOps=1000 many-to-one-helper.h
+-sop=opPut -snumOps=1000 many-to-one-helper.h
+-sop=opFastOn -snumOps=1000 many-to-one-helper.h
+-sop=opOn -snumOps=1000 many-to-one-helper.h
```

Comment:
We can do this later, (separate PR) but I think we'll probably want the correctness test to ensure that the correct  number of gets/puts/AMs occur. We could also add additional checks to ensure the fast-on is actually triggered for opFastOn, and not triggered for opOn.

Reply:
Yep, sounds good and should be straightforward.